### ISSUE TITLE
Build fails on Windows with proj >= 6.2

### ISSUE THREAD

**Issue by QuLogic** [SciTools/cartopy#1460]
> I'm testing a build of 0.18.0b1 on conda-forge as you can see above.  I'm running into trouble with the Windows build, related to:
https://github.com/SciTools/cartopy/blob/master/setup.py#L244-L247
It seems that proj >=6.2 doesn't require this special treatment (the library is just `proj.lib`, though there is a copy as `proj_6_2.lib`) and proj >=6.3 seems to have dropped the numbering entirely (there is just `proj.lib`).  At least this is the case for the packages on `conda-forge`.
> 
> I'm happy to create a separate issue for this if you prefer.

_Originally posted by @xylar in https://github.com/SciTools/cartopy/issues/1458#issuecomment-584513037_

**Comment by xylar** [comment]
Thanks for making this its own issue, @QuLogic.  I believe it will have been addressed once #1463 is merged.
